src/app/app.component.html

```diff
@@ -3,4 +3,51 @@ <h1>
     Najlepsza {{ title }} w sieci!
   </h1>
   <img width="300" alt="Angular Logo" src="http://vitaltrading.pl/wp-content/uploads/2016/04/checklist-1000x675.jpg">
-</div>
\ No newline at end of file
+</div>
+
+
+<input [(ngModel)]="newTask" placeholder="Wpisz zadanie do wykonania" (keyup.enter)='pushTask()' class="addTask">
+<button (click)="pushTask()">Dodaj Zadanie</button>
+<button type='sort' (click)=sortAZ()>Alfabetycznie</button>
+<button type='sort' (click)=sortZA()>Sort od końca</button>
+<button type='sort' (click)=sortNowe()>Najnowsze</button>
```

Comment:
Nawet jeśli interfejs dla użytkownika piszesz po polsku to zawsze pisz kod czyli klasy, metody, zmienne - cokolwiek, po angielsku.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/9cd4748fded56af98574807bd456ab8da35c9095